yazi-adapter/Cargo.toml

```diff
@@ -13,22 +13,22 @@ yazi-config = { path = "../yazi-config", version = "0.3.0" }
 yazi-shared = { path = "../yazi-shared", version = "0.3.0" }
 
 # External dependencies
-ansi-to-tui  = "=3.1.0"
-anyhow       = "1.0.86"
-arc-swap     = "1.7.1"
-base64       = "0.22.1"
+ansi-to-tui  = { workspace = true }
+anyhow       = { workspace = true }
+arc-swap     = { workspace = true }
+base64       = { workspace = true }
 color_quant  = "1.1.0"
-crossterm    = "0.27.0"
-futures      = "0.3.30"
+crossterm    = { workspace = true }
```

Comment:
We can't use `{ workspace = true }` here because the "use-dev-tty" feature can only be enabled on macOS systems.

Reply:
Ah, I see. I hadn't realized. I pushed a fix.